backend/fms_core/models/sample.py

```diff
@@ -60,6 +60,10 @@ def is_depleted(self) -> str:
     def is_pool(self) -> bool:
         return DerivedBySample.objects.filter(sample=self).count() > 1 # More than 1 DerivedBySample implies more than 1 DerivedSample
 
+    @property
+    def is_library(self) -> bool:
+        return True if self.derived_sample_not_pool.library else False
```

Comment:
This could already be coded with pool in mind. In the case of a pool i think we can assume all pooled samples will be libraries, we could test any([derived_sample.library for derived_sample in self.derived_samples]) that should cover the future case of pooled libraries.